plugins/lmgtfy.py

```diff
@@ -7,21 +7,10 @@ class Lmgtfy(BotPlugin):
     question rather than search it for themselves.
     """
 
-    MSG = (
-        'Hey! You seem to have asked a question which could have been solved '
-        'by other means as well. Click [this]({}). When you ask a question, be '
-        'sure to try out a few things first - you\'re in a much better position'
-        ' to help yourself than we are. Try googling, thinking, searching on '
-        'GitHub or [git grep](https://git-scm.com/docs/git-grep) if you are '
-        'looking for source code.'
-        '[StackOverflow](http://stackoverflow.com/help/how-to-ask) '
-        'has a good guide on how to ask questions in a way that allows us to '
-        'help you even faster!'
-    )
-
     @re_botcmd(pattern=r'lmgtfy\s+(.+)',
-               re_cmd_name_help='lmgtfy <search-string>')
+               re_cmd_name_help='lmgtfy <search-string>',
+               template='lmgtfy.jinja2')
     def lmgtfy(self, msg, match):
         """I'm lazy, please google for me."""  # Ignore QuotesBear
         link = 'https://www.lmgtfy.com/?q=' + match.group(1)
-        return self.MSG.format(link)
+        return {'link': link}
```

Comment:
pass in `match.group(1)`, let the template build the link, so it could be changed without code changes.

Also, that would mean that this plugin becomes very similar in purpose and structure to `explain` , only this plugin is for terms which are not (yet) part of the FAQ, overlapping a little with `explain google` in its intention.